TombEngine/Game/Lara/lara_one_gun.cpp

```diff
@@ -1516,7 +1516,7 @@ void HandleProjectile(ItemInfo& item, ItemInfo& emitter, const Vector3i& prevPos
 
 				affectedObjects.push_back(currentItem->Index);
 
-				if (isExplosive)
+				if (isExplosive && !currentObject.undead)
```

Comment:
Does it mean that explosive projectiles won't explode when they will hit undead enemies now?

Reply:
explosive projectiles still explode for undead enemies. It just doesn't damage them and it doesn't explode them